templates/akeneo/files/.platform.app.yaml

```diff
@@ -0,0 +1,102 @@
+# This file describes an application.
+
+# The name of this app. Must be unique within a project.
+name: akeneo
+
+# The toolstack used to build the application.
+type: php:7.2
+
+# Enable non-default extensions required by Akeneo
+runtime:
+    extensions:
+        - apcu
+        - imagick
+
+# The relationships of the application with services or other applications.
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the environment variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+    database: "mysqldb:mysql"
+    elasticsearch: "elasticsearch:elasticsearch"
+
+# The configuration of app when it is exposed to the web.
+web:
+    locations:
+        "/":
+            root: "web"
+            passthru: "/app.php"
+
+# The size of the persistent disk of the application (in MB).
+disk: 3072
+
+# Environment and PHP variables
+variables:
+    php:
+        "date.timezone": "Europe/Paris"
+
+# The mounts that will be performed when the package is deployed.
+# Filesystem is read only by default
+mounts:
+    "var/logs":                 "shared:files/app-logs"
+    "var/cache":                "shared:files/app-cache"
+    "var/uploads":              "shared:files/app-uploads"
+    "app/file_storage":         "shared:files/app-file_storage"
+    "app/archive":              "shared:files/app-archive"
+    "app/uploads/product":      "shared:files/app-uploads"
+    "installer":                "shared:files/installer"
+    "web/media":                "shared:files/media-cache"
+
+hooks:
+    # We run build hooks before your application has been packaged.
+    build: |
+        # install nvm to use newer version of node
+        unset NPM_CONFIG_PREFIX
+        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | dash
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+        # install node 10 (currently latest LTS)
+        nvm install 10
+
+        # install yarn
+        npm install -g yarn
+
+        # run yarn to install frontend
+        yarn install
+        yarn run webpack
+
+        # remove dev file
+        rm web/app_dev.php
+
+    # We run deploy hook after your application has been deployed and started.
```

Comment:
"We run the deploy hook...".  Probably a pre-existing bug we should fix elsewhere, too.

Reply:
changed (also in the build hook)